docker-compose.yml

```diff
@@ -13,6 +13,18 @@ services:
     volumes:
       - ./vendor/prooph/pdo-event-store/scripts/mysql:/docker-entrypoint-initdb.d
 
+#  postgres:
```

Comment:
why comment out?

Reply:
Because MySql container is used by default